src/common/meta/src/ddl_manager.rs

```diff
@@ -457,6 +498,39 @@ async fn handle_create_table_task(
     Ok(SubmitDdlTaskResponse {
         key: id.to_string().into(),
         table_id: Some(table_id),
+        ..Default::default()
+    })
+}
+
+async fn handle_create_logical_table_tasks(
+    ddl_manager: &DdlManager,
+    cluster_id: u64,
+    mut create_table_tasks: Vec<CreateTableTask>,
+) -> Result<SubmitDdlTaskResponse> {
+    ensure!(!create_table_tasks.is_empty(), EmptyCreateTableTasksSnafu);
+    let physical_table_info = ddl_manager
+        .table_metadata_allocator
+        .set_table_ids_on_logic_create(&mut create_table_tasks)
+        .await?;
+    let physical_table_id = physical_table_info.0;
+    let num_logical_tables = create_table_tasks.len();
+
+    let id = ddl_manager
+        .submit_create_logical_table_tasks(
+            cluster_id,
+            create_table_tasks,
+            physical_table_info,
+            HashMap::new(),
+        )
+        .await?;
+
+    info!("{num_logical_tables} logical tables on physical table: {physical_table_id:?} is created via procedure_id {id:?}");
+
+    Ok(SubmitDdlTaskResponse {
+        key: id.to_string().into(),
+        // TODO(jeremy): add table_ids to response
```

Comment:
Waiting for this PR:https://github.com/GreptimeTeam/greptimedb/pull/3201

Reply:
It's merged.

Reply:
done